README.md

```diff
@@ -37,12 +36,10 @@ Developing Mayu requires the following tools to be installed.
 
 ## Getting Mayu
 
-Download the latest release from here: https://github.com/giantswarm/mayu/releases/latest
+Download the latest release from here: get latest docker image here: https://quay.io/repository/giantswarm/mayu
```

Comment:
I'd just go with 

`Get the latest Docker image here: https://quay.io/repository/giantswarm/mayu`